programs/zstdcli.c

```diff
@@ -756,6 +756,7 @@ int main(int argCount, const char* argv[])
                     if (longCommandWArg(&argument, "--stream-size=")) { streamSrcSize = readU32FromChar(&argument); continue; }
                     if (longCommandWArg(&argument, "--target-compressed-block-size=")) { targetCBlockSize = readU32FromChar(&argument); continue; }
                     if (longCommandWArg(&argument, "--size-hint=")) { srcSizeHint = readU32FromChar(&argument); continue; }
+                    if (longCommandWArg(&argument, "--output-dir-flat=")) { outDirName = argument; continue; }
```

Comment:
I'm confused. How is this different from the existing test above (line 692)? Should the other still be present with this added here?

Reply:
Ohhhh, I see, this is if it's provided as a single arg in `--option=val` style.

Reply:
They are both enabled.
`--output-dir-flat DIR` is just equivalent to `--output-dir-flat=DIR`.

I tend to prefer the second formulation, which is only possible with long commands.